### PR TITLE
refactored readme WIP

### PR DESCRIPTION
**Pull request by mandric**
WIP https://github.com/medic/medic-webapp/issues/3053

### CODE REVIEW

**Comment by mandric** [comment]
Not ready for review yet.

File: README.md [hunk 1]
```diff
@@ -52,68 +49,57 @@ Some CDMA networks have limited support for multipart SMS messages.  This can oc
...
 	}
 
 ### POST
 
-`medic-gateway` will accept and process any relevant data received in a response.  However, it may choose to only send certain types of information in a particular request (e.g. only provide a webapp-terminating SMS), and will also poll the web service periodically for webapp-originating messages, even if it has no new data to pass to the web service.
+Using the POST verb `medic-gateway` supports the following:
 
-### Request
+ - accepts and processes relevant data received in a response
+ - provides the web service with server-terminiating SMS data
```
Comment (estellecomment) [inline README.md]: typo : terminating

File: README.md [hunk 2]
```diff
@@ -52,68 +49,57 @@ Some CDMA networks have limited support for multipart SMS messages.  This can oc
...
-Where a list of values is expected but there are no values provided, it is acceptable to:
+# HTTP API
 
-* provide a `null` value; or
-* provide an empty array (`[]`); or
-* omit the field completely
+This is the API specification for communications between `medic-gateway` and a web service.  Request and response bodies in both directions are formatted in JSON.  
 
-Bar array behaviour specified above, `medic-gateway` _must_ include fields specified in this document, and the web server _must_ include all expected fields in its responses.  Either party _may_ include extra fields as they see fit.
+The the entire API should be implemented by a web service at a single endpoint, e.g. https://example.com/api/sms
```
Comment (estellecomment) [inline README.md]: typo "The the"

**Comment by garethbowen** [comment]
Closing as ancient.

**Pull request closed without merging**